src/utils/Rules.js

```diff
@@ -1,12 +1,11 @@
-import axios from 'axios';
 import { stringify } from 'qs';
+import axios from './axios';
 
-const baseUrl = `${process.env.BALROG_ROOT_URL}/api`;
-const getRules = () => axios.get(`${baseUrl}/rules`);
-const getChannels = () => axios.get(`${baseUrl}/rules/columns/channel`);
-const getProducts = () => axios.get(`${baseUrl}/rules/columns/product`);
+const getRules = () => axios.get('/rules');
+const getChannels = () => axios.get('/rules/columns/channel');
+const getProducts = () => axios.get('/rules/columns/product');
 const getHistory = (id, limit, page) =>
-  axios.get(`${baseUrl}/${id}/revisions?${stringify({ limit, page })}`);
+  axios.get(`/${id}/revisions?${stringify({ limit, page })}`);
```

Comment:
I think latest master has some of these rules use `baseUrl`. We would probably need to rebase and fix those functions. Example: https://github.com/mozilla-frontend-infra/balrog-ui/blob/68b3562949c8e947991b7a3671ecb82a78d07f0e/src/utils/Rules.js#L15-L23.